internal/prefixcollector/collector.go

```diff
@@ -116,8 +119,23 @@ func (epc *ExcludedPrefixCollector) Serve(ctx context.Context) {
 	}
 }
 
+// splitPrefix splits single prefix string into list of prefixes treating the input as comma separated.
+// When cluster supports both IPv4 and IPv6 we can receive combined addresses e.g. "10.244.0.0/16,fd00:10:244::/56"
+func splitPrefix(prefix string) []string {
+	raws := strings.Split(prefix, ",")
+	parts := make([]string, 0, len(raws))
```

Comment:
Note: in go you can get len of nil slice.


```suggestion
	var parts []string
```